### ISSUE TITLE
Pipeline scheduled to run after pipeline executed only once - first time the depending pipeline finishes

### ISSUE THREAD

**Issue by eea03** [UnifiedViews/Core#486]
When a pipeline is scheduled to run after some other pipeline, this pipeline is executed only the first time the depedning pipeline runs. Afterwards this schedule rule is disabled automatically.

One would expect that pipeline scheduled after another pipeline should run always when the other pipeline ends, not just the first time.

Maybe this should be configurable as for periodic schedules, but still by default I would expect that by default "after pipeline schedule" should be executed always, not just once

**Comment by eea03** [comment]
Related also to https://github.com/UnifiedViews/Core/issues/432

**Comment by tomas-knap** [comment]
True. I will examine what is the required behaviour, but I agree, more logical would be if such schedulling rule will work forever, until it is manually switched off.
